examples/src/main/scala/com/gu/tableversions/examples/TableLoader.scala

```diff
@@ -6,7 +6,7 @@ import cats.effect.IO
 import com.gu.tableversions.core.TableVersions.{UpdateMessage, UserId}
 import com.gu.tableversions.core.{TableDefinition, TableVersions, Version}
 import com.gu.tableversions.metastore.Metastore
-import com.gu.tableversions.spark.VersionedDataset
+import com.gu.tableversions.spark.VersionContext
```

Comment:
I think you can remove some `import`s here now.

Reply:
Who needs `-Ywarn-unused:imports` with eyes like that? 😀